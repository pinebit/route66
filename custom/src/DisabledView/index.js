import React from 'react';
import { Segment, Header } from 'semantic-ui-react';

const DisabledView = () => (
  <Segment textAlign="center" padded="very" piled>
    <Header as="h2" color="red">
            Your account is disabled. Please contact any Manager to enable your account.
    </Header>
  </Segment>
);

export default DisabledView;
