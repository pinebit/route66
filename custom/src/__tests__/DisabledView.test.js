import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DisabledView from '../DisabledView';

describe('<DisabledView />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<DisabledView />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
