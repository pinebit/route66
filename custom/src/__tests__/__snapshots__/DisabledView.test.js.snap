// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisabledView /> renders correctly 1`] = `
<Segment
  padded="very"
  piled={true}
  textAlign="center"
>
  <Header
    as="h2"
    color="red"
  >
    Your account is disabled. Please contact any Manager to enable your account.
  </Header>
</Segment>
`;
